src/primitive/time_evolution.jl

```diff
@@ -6,6 +6,10 @@ export TimeEvolution, time_evolve
     TimeEvolution{N, TT, GT} <: PrimitiveBlock{N, ComplexF64}
 
 TimeEvolution, where GT is block type. input matrix should be hermitian.
+
+Note:
```

Comment:
this is how you write note in docs

```suggestion
!!!note:
```